tests/conftest.py

```diff
@@ -141,13 +141,15 @@ def pytest_configure(config):
 
 
 @pytest.fixture
-def dummy_stage(tmp_dir, dvc):
-    def make(path="dvc.yaml", name="dummy_stage", **kwargs):
-        from dvc.stage import PipelineStage, create_stage
+def make_stage(tmp_dir, dvc):
```

Comment:
@pared, we might not need this fixture at all if we decide on `dvc.stage.add()` (just need to type a bit more though).
The only thing that needs to be decided is whether it fits into the current `dvc.stage` or should it be a different part of the abstraction (`dvc.index`?).

Reply:
makes sense, do we have a discussion for that somewhere?